# HG changeset patch
# User Simon King <simon.king@uni-jena.de>
# Date 1354202289 -3600
# Node ID 88cfad9e089148694276da1d8a605bf4ee015054
# Parent  ace49ff24a907b119b4aa274ab81a5abc8313825
See Sage ticket #13731, Singular ticket 463: Fix out-of-bound error in facFactorize.cc

diff --git a/factory/facFactorize.cc b/factory/facFactorize.cc
--- a/factory/facFactorize.cc
+++ b/factory/facFactorize.cc
@@ -705,7 +705,7 @@
 
     evaluationWRTDifferentSecondVars (bufAeval2, bufEvaluation, A);
 
-    for (int j= 0; j < A.level() - 1; j++)
+    for (int j= 0; j < A.level() - 2; j++)
     {
       if (!bufAeval2[j].isEmpty())
         counter++;
